### PR TITLE
add com.endlessnetwork.fablemaker

### PR DESCRIPTION
**Pull request by alexthemighty**
submission of Fablemaker app. 
If approved please grant repository access to nan@endlessnetwork.com as well.
Thanks!

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.endlessnetwork.fablemaker

File: com.endlessnetwork.fablemaker.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "strip" : true,
+    "no-debuginfo" : true
+  },
+  "command" : "com.endlessnetwork.fablemaker.sh",
+  "modules" : [
+    { 
+        "name" : "Fablemaker",
+        "buildsystem" : "simple",
+        "only-arches" : ["x86_64"],
+	"sources" :
```
Comment (nedrichards) [inline com.endlessnetwork.fablemaker.json]: can you fix the indenting on this file? it's a little hard to read. run it through a linter like jq

File: com.endlessnetwork.fablemaker.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "install -Dm644 FM_Favicon_128x128.png /app/share/icons/hicolor/128x128/apps/com.endlessnetwork.fablemaker.png",
+        "install -Dm644 FM_Favicon_64x64.png /app/share/icons/hicolor/64x64/apps/com.endlessnetwork.fablemaker.png",
+        "install -D com.endlessnetwork.fablemaker.sh /app/bin/com.endlessnetwork.fablemaker.sh",
+        "mv Fablemaker_Data /app/bin/",
+        "install Fablemaker.x86_64 /app/bin/Fablemaker.x86_64"
+        
+      ]
+    }
+      ],
+  "separate-locales" : false,
```
Comment (nedrichards) [inline com.endlessnetwork.fablemaker.json]: we normally like these fields at the top of the file as it's a bit easier to reason about in review.

**Comment by nedrichards** [comment]
other than the minor formatting nitpick lgtm.

**Comment by LeandroStanger** [comment]
bot, build com.endlessnetwork.fablemaker

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.endlessnetwork.fablemaker

**Comment by nedrichards** [comment]
@alexthemighty nan@endlessnetwork.com doesn't seem to have a github account, let me know what the username is and I'll add it.

**Comment by LeandroStanger** [comment]
@nedrichards username @EndlessNaN

**Comment by alexthemighty** [comment]
> @nedrichards username @EndlessNaN

yeah, sorry I assumed it was tied to that email, but this should be correct.  Thanks for letting him know!

**Comment by nedrichards** [comment]
invitation sent.

**Pull request closed without merging**